tests/openwisp2/settings.py

```diff
@@ -178,13 +183,7 @@
         'description': 'Histogram',
         'top_fields': 2,
         'order': 999,
-        'query': {
-            'influxdb': (
-                "SELECT {fields|SUM|/ 1} FROM {key} "
-                "WHERE time >= '{time}' AND content_type = "
-                "'{content_type}' AND object_id = '{object_id}'"
-            )
-        },
+        'query': query[TIMESERIES_DB['DATABASE']]['histogram'],
```

Comment:
As the above comment says, this is used ofr automated testing purposes. If we support different DBs we need to test them all, keep this is mind.

Reply:
@PabloCastellano you're right, one of the measurable outcomes is to run the test with both timeseries DB